### PR TITLE
Add Performance improvement

### PR DESCRIPTION
**Pull request by yash244466666**
In this project, We implemented the following:
- Optimized some slow queries in my database.

Screenshot (93)
Screenshot (94)
Screenshot (95)
Screenshot (96)
Screenshot (97)
Screenshot (98)
Screenshot (99)
Screenshot (100)
Screenshot (101)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -55,13 +55,45 @@ CREATE TABLE vets (
...
+SELECT * FROM owners where email = 'owner_18327@mail.com';
+
+-- Use EXPLAIN ANALYZE on the previous queries to check what is happening. Take screenshots of them -
+
+-- before improve
+EXPLAIN ANALYSE SELECT COUNT(*) FROM visits where animal_id = 4;
+SELECT COUNT(*) FROM visits where animal_id = 4;
+
+-- after improve
+CREATE INDEX visits_animal_id ON visits(animal_id);
```
Comment (franclobo) [inline schema.sql:81]: - [x] The planning and execution time should be shorter than before creating the index, I suggest using ASC or DESC in the id column to reduce the times for good optimization. 👍🏽

File: schema.sql [hunk 2]
```diff
@@ -55,13 +55,45 @@ CREATE TABLE vets (
...
+EXPLAIN ANALYSE SELECT COUNT(*) FROM visits WHERE animal_id = 4;
+
+-- SELECT * FROM visits where vet_id = 2; Before improve
+SELECT * FROM visits WHERE vet_id = 2;
+explain analyse SELECT COUNT(*) FROM visits where animal_id = 2;
+
+-- Querie : SELECT * FROM visits where vet_id = 2; After improve
+CREATE INDEX visits_vet_id ON visits(vet_id);
```
Comment (franclobo) [inline schema.sql:89]: - [x] The planning and execution time should be shorter than before creating the index, I suggest using ASC or DESC in the id column to reduce the times for good optimization. 👍🏽

File: schema.sql [hunk 3]
```diff
@@ -55,13 +55,45 @@ CREATE TABLE vets (
...
+EXPLAIN ANALYSE SELECT * FROM visits WHERE animal_id = 2;
+
+-- Querie :SELECT * FROM owners where email = 'owner_18327@mail.com';
+-- Before
+EXPLAIN ANALYSE SELECT * FROM owners WHERE email ='owner_18327@mail.com';
+SELECT FROM owners WHERE email='owner_18327@mail.com';
+
+-- after
+CREATE INDEX owners_email ON owners(email);
```
Comment (franclobo) [inline schema.sql:98]: - [x] The planning and execution time should be shorter than before creating the index, I suggest using ASC or DESC in the id column to reduce the times for good optimization. 👍🏽

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @yash244466666 and @tonnymuchui, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- All queries created ✔️ 
- Explain Analyze command used 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] There should be ✋🏽 ☝🏽  six pictures at most one before and one after creating indexes for each query, please paste the screenshots one after another to order for good comprehension for anybody who looks at your PR.
- [ ] I recommend altering 👽 the table owner to add a column email for creating the corresponding query.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by yash244466666** [comment]
-  **Added ASC  to reduce the times for good optimization.**

Screenshot (102)
Screenshot (103)
Screenshot (104)

**Comment by arslanbisharat** [comment]
Hi @yash244466666,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**